components/squarespace/sources/new-create-transaction/new-create-transaction.mjs

```diff
@@ -0,0 +1,30 @@
+import squarespace from "../../squarespace.app.mjs";
+
+export default {
+  name: "New Create Transaction",
+  version: "0.0.1",
+  key: "squarespace-new-create-transaction",
+  description: "Emit new event for each transaction created.",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    squarespace,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 5,
+      },
+    },
+  },
+  async run() {
+    const transactions = await this.squarespace.getTransactions();
```

Comment:
As in `new-create-product`, is it returning all transactions or new transactions?

Reply:
Just "new" transctions, the polling way first return all transactions, to after return just new.